src/api/v1/notifications/handlers/handle-xmtp-notification.ts

```diff
@@ -62,6 +69,10 @@ export async function handleXmtpNotification(req: Request, res: Response) {
       return;
     }
 
+    const primaryIdentity = device.identities[0].identity;
```

Comment:
I don't see how a device can have fewer than 1 identity, but would be nice to be defensive here anyways and handle the case.